up42/workflow.py

```diff
@@ -100,47 +100,69 @@ def _construct_full_workflow_tasks_dict(
     ) -> List[Dict]:
         """
         Constructs the full workflow task definition from a simplified version.
```

Comment:
Maybe one important note to add is that if you call blocks by name, the version will always be the latest, while if you call the block id, you can essentially "pin" to a block version.